docs/PacketFence_Clustering_Guide.asciidoc

```diff
@@ -1157,6 +1159,7 @@ At the end it you want to check the status of the slave server for debug purpose
 
  SHOW SLAVE STATUS;
 
+HAProxy daemon is responsible to detect when link between DC2 and DC1 is down.
```

Comment:
Its a been random to put this here without any other context around it

Reply:
A complete reorganization of this section is necessary to have something clear.

I prefer to have information even if it's not perfect.

Reply:
At least make this clearer by saying its haproxy-db and talking explicitly about the database.

This sentence just looks like it was randomly dropped into the document and just looks odd.
Kinda like if @lzammit would go to a yoga class.

Reply:
Is it better like this ?